vignettes/momentumandusedata.Rmd

```diff
@@ -121,6 +121,9 @@ if (rebuild) {
     lambda = 0, batches = 0.1, label = labels)
 	for (batches in c(0.3, 0.8)) {
 		newCoords <- data.frame(t(projectKNNs(preVis$wij, 
+		verbose = TRUE, 
```

Comment:
Probably should not be setting verbose=T in vignette.

Reply:
I'll change to verbose=FALSE